Exiled.API/Features/Player.cs

```diff
@@ -1222,23 +1222,58 @@ public void ShowHint(string message, float duration = 3f)
         }
 
         /// <summary>
-        ///  Disables all status affects on the player.
+        /// Gets an array of active status effects on the player.
+        /// </summary>
+        /// <returns>A <see cref="PlayerEffect"/> array indicating active status effects.</returns>
+        public PlayerEffect[] GetActiveEffects()
+        {
+            PlayerEffect[] arr = new PlayerEffect[] { };
+            foreach (KeyValuePair<Type, PlayerEffect> data in ReferenceHub.playerEffectsController.AllEffects)
+            {
+                if (data.Value.Enabled)
+                {
+                    arr.Append(data.Value);
+                }
+            }
+
+            return arr;
+        }
```

Comment:
Is it really useful? It literally allocates N arrays, where N is the number of enabled effects. I don't find it so useful.

Reply:
fair enough